recipe/build_fftw.sh

```diff
@@ -70,9 +70,15 @@ if [[ "$target_platform" == "linux-aarch64" ]]; then
 fi
 
 if [[ "$target_platform" == "osx-arm64" ]]; then
-  ARCH_OPTS_SINGLE="--enable-neon"
-  ARCH_OPTS_DOUBLE="--enable-neon"
-  DISABLE_LONG_DOUBLE=1
+  # Disable neon for now (issue # 94).  See https://github.com/FFTW/fftw3/issues/129
+  # ARCH_OPTS_SINGLE="--enable-neon --enable-armv8-cntvct-el0"
+  # ARCH_OPTS_DOUBLE="--enable-neon --enable-armv8-cntvct-el0"
+  #
+  # Add cycle counter: https://www.fftw.org/fftw3_doc/Cycle-Counters.html
+  # Run ./configure --help in the fftw source dir for possible flags. 
+  ARCH_OPTS_SINGLE="--enable-armv8-cntvct-el0"
+  ARCH_OPTS_DOUBLE="--enable-armv8-cntvct-el0"
+  DISABLE_LONG_DOUBLE=""
```

Comment:
Unfortunately, `long double` is the same as `double` on osx-arm64 (see e.g. https://discourse.llvm.org/t/support-for-128bit-long-double-on-apple-silicon/66108/4), so this should probably stay disabled.

Reply:
Thanks for the information. I have disabled it and included a reference to the official docs.

Reply:
Perfect, thanks!